com.voxdsp.PoryDrive.yaml

```diff
@@ -0,0 +1,40 @@
+app-id: com.voxdsp.PoryDrive
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: porydrive
+rename-icon: porydrive
+rename-desktop-file: porydrive.desktop
+rename-appdata-file: porydrive.appdata.xml
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+modules:
+  - name: glfw
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBUILD_SHARED_LIBS=ON
+      - -DGLFW_BUILD_EXAMPLES=OFF,
+      - -DGLFW_BUILD_TESTS=OFF
+      - -DGLFW_BUILD_DOCS=OFF
+      - -DGLFW_USE_WAYLAND=1
+    sources:
+      - type: archive
+        url: https://github.com/glfw/glfw/releases/download/3.3.8/glfw-3.3.8.zip
+        commit: 7482de6071d21db77a7236155da44c172a7f6c9e
```

Comment:
there is no commit for an archive (The error is misleading). what you need is a sha256 of the archive